<launch>

  <!--frequency [Hz]-->

  <arg name="frequency" default="30" />
  <arg name="control_frequency" default="30" />


  <!--rogi_node-->

  <node pkg="path_planning_tr" name="path_planning" type="path_planning_node" output="screen">
    <param name = "frequency" value = "frequency"/>
  </node>


  <!--joystick-->

  <include file="$(find joystick_tr)/launch/joystick_tr.launch">
  <arg name="frequency" value="$(arg control_frequency)" />
  </include>


  <!--mbed-->

  <node pkg="serial_test" name="mbed_arm" type="serial_test_node" output="screen">
    <param name = "port" value = "/dev/serial/by-id/usb-STMicroelectronics_STM32_STLink_066EFF555054877567044940-if02"/>
    <remap from = "/Serial_sub_int" to = "arm_mbed" />
  </node>

</launch>
